tools/flowc/incremental.flow

```diff
@@ -25,7 +25,7 @@ export {
 	mergeIncremental2TypeEnvProgram(env : FcTypeEnvProgram, globEnv : FcTypeEnvGlobal, module : FiModule) -> FcTypeEnvProgram;
 }
 
-incrementalVersion = 6;
+incrementalVersion = 7;
```

Comment:
What is new in the object files?

Reply:
I added a new field to FiFileInfo struct:
`mutable source : string, // is loaded by demand. Initially is an empty string.`
because hover feature may demand frequent accesses to the source code (as a string), so it's better to have it cached in memory.